src/components/LessonHighlights/LoggedInUserLessonsList.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react"
+import styled from "styled-components"
+import { ContentfulLesson } from "../../../graphql-types"
+import LessonCard from "../lesson/LessonCard"
+import { listLikedContents } from "../../graphql/queries"
+import { API, graphqlOperation } from "aws-amplify"
+import { useQuery } from "react-query"
+
+interface Props {
+  data: any
+}
+
+const LoggedInUserlessonsList = (props: Props) => {
+  const fetchLikes = async () => {
+    return await API.graphql(graphqlOperation(listLikedContents))
```

Comment:
It came to my mind, that you can also alter the grahql query so that you only fetch likes for content type that is week etc, using the graphql filter function